### PR TITLE
[ZEPPELIN-5898]fix:down csv data error

### PR DESCRIPTION
**Pull request by zhugezifang**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - https://zeppelin.apache.org/contribution/contributions.html

### What type of PR is it?
Bug Fix
Improvement
Feature
Documentation
Hot Fix
Refactoring
*Please leave your type of PR only*

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN/
* Put link here, and add [ZEPPELIN-*Jira number*] in PR title, eg. [ZEPPELIN-533]

### How should this be tested?
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the license files need to update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

File: zeppelin-web/src/app/notebook/paragraph/result/result.controller.js [hunk 1]
```diff
@@ -895,6 +895,9 @@ function ResultCtrl($scope, $rootScope, $route, $window, $routeParams, $location
           if (row.hasOwnProperty(index)) {
             let stringValue = (row[index]).toString();
             if (stringValue.indexOf(delimiter) > -1) {
+              if (stringValue.indexOf('"') > -1) {//判断json字符串
```
Comment (Reamer) [inline zeppelin-web/src/app/notebook/paragraph/result/result.controller.js:898]: Please write comments in English.

**Review by Reamer — CHANGES_REQUESTED** [review]
Does this problem also exist in the new frontend?

**Comment by zjffdu** [comment]
Thanks for the contribution @zhugezifang , could you update the PR description?

**Comment by zhugezifang** [comment]
> Thanks for the contribution @zhugezifang , could you update the PR description?

ok thanks, i updated it

File: zeppelin-web/src/app/notebook/paragraph/result/result.controller.js (hunk 1 above)
Comment (zhugezifang) [reply zeppelin-web/src/app/notebook/paragraph/result/result.controller.js]: > Please write comments in English.

ok, i am sorry, i updated it

**Pull request closed without merging**